cmd/hw_02/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"golang-course/cmd/hw_02/fibonacci/dp"
+	"golang-course/cmd/hw_02/fibonacci/recursive"
+)
+
+func main() {
+	fmt.Println("Enter n, amount of Fibonacci nums. Range: 1-93")
+	var n int
+	fmt.Scanf("%d\n", &n)
+
+	if n < 1 || n > 93 {
+		fmt.Println("Incorrect value")
+	} else {
+		fmt.Println("Choose type of solution: 1-dynamic programming, 2-recursion")
```

Comment:
Please read https://golang.org/doc/effective_go#if.  Avoid using "else" unless necessary. 
```suggestion
	if n < 1 || n > 93 {
		fmt.Println("Incorrect value")
		return
	} 
	fmt.Println("Choose type of solution: 1-dynamic programming, 2-recursion")
	
```